docs/themes/manon-docs/layouts/partials/section-index.html

```diff
@@ -8,7 +9,7 @@
 
     {{ range $pages }}
     <p>
-    <h2>
+    <h2 id="{{ .Title | plainify | htmlUnescape | anchorize }}">
```

Comment:
I think using the `anchorize` filter makes the other filters unnecessary since it already sanitizes the input